### DISCUSSION TITLE
Необходимость dev-ветки

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#607] [Идеи] [answered]
Необходимо решить, нужна ли нам промежуточная dev-ветка для тестирования нового функционала и PR внешних и внутренних контрибьюторов.

## Флоу

- У проекта есть две ветки: dev и main
- Все нововведения платформы, кроме хотфиксов вливаются в dev-ветку, тестируются там, затем вливаются в main.
- Хотфиксы вливаются прямо в main

## Плюсы

- Увеличивается стабильность сборки, в production попадает значительно меньше ошибок

## Минусы

- Усложняется флоу

**Comment by pepelsbey** [comment]
К счастью, речь исключительно про платформу. То есть какой-то уровень сложности мы можем себе позволить.

Но даже при том, что это только платформа, я был бы рад, если бы пул-реквест можно было делать очень просто, а уже автоматизация меняла целевую ветку, если это возможно. Что мы, собственно, и обсудили.

**Comment by furtivite** [comment, marked as answer]
Дубль #284
